tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -143,6 +155,23 @@ echo "$BAZEL_LABEL,$BAZEL_TARGET_ID" >> "$SCHEME_TARGET_IDS_FILE"
             environmentBuildable: buildableReference
         )
     }
+    
+    /// Symlinks `$DEVELOPER_DIR/Toolchains/XcodeDefault.xctoolchain/usr/lib` to `$(BAZEL_INTEGRATION_DIR)/lib` so that Xcode can copy sanitizers' dylibs.
+    static func symlinkDefaultToolchainUsrLibDirectory(
+        buildableReference: XCScheme.BuildableReference
+    ) -> XCScheme.ExecutionAction {
+        return .init(
+            scriptText: #"""
+if [ "${ENABLE_THREAD_SANITIZER:-}" == "YES" ]; then
+    dest="$BAZEL_INTEGRATION_DIR/lib"
+    DEV_LIB_DIR="$SYSTEM_DEVELOPER_DIR/Toolchains/XcodeDefault.xctoolchain/usr/lib"
```

Comment:
Why the use of `SYSTEM_DEVELOPER_DIR` versus `DEVELOPER_DIR`?

Reply:
I found that the Scheme script env had `SYSTEM_DEVELOPER_DIR` that was pointing to $(xcode-select -p) and looks like `DEVELOPER_DIR ` was not there. I will double-check on this and will let you know.

Reply:
`DEVELOPER_DIR` is present - my bad. I will update the PR. Thanks!

Reply:
Updated!